data/json/furniture_and_terrain/furniture-surfaces.json

```diff
@@ -66,7 +66,7 @@
     "description": "A commercial-quality swinging door made of wood that allows passage behind counters.  It's currently closed.",
     "symbol": "+",
     "color": "blue",
-    "move_cost_mod": 2,
+    "move_cost_mod": 0,
```

Comment:
```suggestion
    "move_cost_mod": 2,
```

Closed counter door should still have a movement penalty

Reply:
I'm not making it crossable so that's irrelevant

Reply:
Yes, but tooltip is still shown. Could be confusing if it is different from a regular counter

Reply:
Why would it be confusing?

Reply:
When you inspect it with `X`, it would say that movement cost is 100, like it is not there.

So if there is no difference between open and closed gate, why open it then :)